client/src/components/graph/graph.tsx

```diff
@@ -306,62 +308,58 @@ class Graph extends React.Component<{}, GraphState> {
         tool correctly reflects the underlying selection.
         */
     if (
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'currentSelection' does not exist on type... Remove this comment to see the full error message
       currentSelection !== prevProps.currentSelection ||
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'graphInteractionMode' does not exist on ... Remove this comment to see the full error message
       graphInteractionMode !== prevProps.graphInteractionMode ||
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'toolSVG' does not exist on type '{}'.
       stateChanges.toolSVG
     ) {
       const { tool, container } = this.state;
       this.selectionToolUpdate(
-        // @ts-expect-error ts-migrate(2339) FIXME: Property 'tool' does not exist on type '{}'.
-        stateChanges.tool ? stateChanges.tool : tool,
-        // @ts-expect-error ts-migrate(2339) FIXME: Property 'container' does not exist on type '{}'.
-        stateChanges.container ? stateChanges.container : container
+        stateChanges.tool ? stateChanges.tool : tool!,
```

Comment:
There's a lot of non-null assertions because state is initiated as null, but quickly setup up after mounting